src/menus/icon-menu.jsx

```diff
@@ -133,7 +133,7 @@ const IconMenu = React.createClass({
       ref: this.state.iconButtonRef,
     });
 
-    let menu = open ? (
+    let menu = open && this.props.children.length > 0 ? (
```

Comment:
So, if we are not passing children to this component it's like using a simple `IconButton`.
Instead, I think that we should throw a console.error to the developer in the development environment.